openassessment/xblock/submission_mixin.py

```diff
@@ -435,8 +435,10 @@ def submission_path_and_context(self):
             student_submission = self.get_user_submission(
                 workflow["submission_uuid"]
             )
-            context["peer_incomplete"] = "peer" in workflow["status_details"] and not workflow["status_details"]["peer"]["complete"]
-            context["self_incomplete"] = "self" in workflow["status_details"] and not workflow["status_details"]["self"]["complete"]
+            peer_in_workflow = "peer" in workflow["status_details"]
+            self_in_workflow = "self" in workflow["status_details"]
+            context["peer_incomplete"] = peer_in_workflow and not workflow["status_details"]["peer"]["complete"]
+            context["self_incomplete"] = self_in_workflow and not workflow["status_details"]["self"]["complete"]
```

Comment:
Would these be "cleaner" with `get()` and default values?

`context["peer_incomplete"] = workflow["status_details"].get("peer"), {}).get("complete", False)`

I'm okay with this as it is here, just throwing a suggestion out there.